src/main/java/core/modules/web/models/WebAssertion.java

```diff
@@ -58,6 +54,31 @@ public void shouldBeDisplayed(boolean condition) {
         Assert.assertFalse(elementForAssert.isDisplayed(), "Element " + elementForAssert.toString() + " is displayed");
     }
 
+    public void shouldBeVisibleByCss(boolean condition, String cssSelector) {
```

Comment:
Better to use obvious names: shouldBeVisibleByCss and shouldNotBeVisibleByCss

Reply:
I agree. I'll delete this method for now.